requirements-bddtests.in

```diff
@@ -0,0 +1,7 @@
+httpretty
+pytest
+factory-boy
+h-matchers
+webtest
+behave
+-r requirements.txt
```

Comment:
There's some duplication between the `requirements-*.in` files, particularly the different `requirements-*tests.in` files. I think this is fine: we don't want to complicate things by making all these files depend on a base file or something. And we might actually want to pin a dependency to < or > some version for the functests but not the bddtests, for example. Let's just keep it simple and leave the duplication